modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -1215,6 +1227,50 @@ private Entry doGet(byte[] key, long revUpperBound) {
         return doGetValue(key, lastRev);
     }
 
+    /**
+     * Gets the value by key and revisions.
+     *
+     * @param key            Target key.
+     * @param revLowerBound  Target lower bound of revision.
+     * @param revUpperBound  Target upper bound of revision.
+     * @return Value.
```

Comment:
Just value? It's a list of revision based values ordered by revisions, isn't it. Please also address it in
`Gets the value by key and revisions.`.

Reply:
I've updated javadoc. I tried to describe it more clearly. Not sure if I succeeded. But I can't write better.